base/div.jl

```diff
@@ -133,15 +133,29 @@ julia> divrem(7,3)
 ```
 """
 divrem(x, y) = divrem(x, y, RoundToZero)
+
+function divrem(a::T, b::T, r::RoundingMode) where {T<:AbstractFloat}
```

Comment:
```suggestion
function divrem(a::AbstractFloat, b::AbstractFloat, r::RoundingMode)
```

Otherwise the method below will be called for different-type floats, which I think we don't want?

Reply:
I wanted to write it for Float16, Float32, Float64, BigFloat - all the subtypes of AbstractFloat. I don't know what the difference will be